sdk/Lusid.Sdk.Tests/LusidApiFactoryTests.cs

```diff
@@ -107,7 +107,29 @@ public void InvalidApiUrl_ThrowsException()
                 () => new LusidApiFactory(apiConfig),
                 Throws.InstanceOf<UriFormatException>().With.Message.EqualTo("Invalid LUSID Uri: xyz"));
         }
-        
+
+        [Test]
+        public void NetworkConnectivityErrors_ThrowsException()
```

Comment:
Great test, and looks good overall

Do we want to add some extra tests checking what happens on regular errors >400 errors  just for sanity?

Reply:
I've now pulled over the LusidExceptionHandlerTests, so we've got tests like `CallCustomExceptionFactory_FailsWithApiError_ReturnsTheSameErrorCodeAsApi` (testing 404, for example)